src/styles/calendar.scss

```
@@ -0,0 +1,61 @@
+$month-length: 29;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 800px;
+  gap: 1px;
+  padding: 10px;
+}
+
+.day {
+  @for $index from 1 through 31 {
+
+    &--#{$index}::before {
+      content: "#{$index}";
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: black;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+  }
+
+  $start-day:
+    "mon" 0,
+    "tue" 101px,
+    "wed" 202px,
```

What is 101? 102? Use variables

```suggestion
    "tue" day-width + gap,
    "wed" day-width * 2 + gap * 2,
```